composer.json

```diff
@@ -24,7 +24,8 @@
   ],
   "require": {
     "php": ">=5.4.0",
-    "yiisoft/yii2": "^2.0.0"
+    "yiisoft/yii2": "^2.0.0",
+    "npm-asset/fortawesome--fontawesome-free": "~5.7.1"
```

Comment:
Not sure if unconditional installation of the package is appropriate. Many people use the pro version, they do not need free.

Reply:
Ok I'll remove it and update docs